### PR TITLE
Clarify command line help for report --run_results & --run_matrices

### PR DESCRIPTION
**Pull request by peterjc**
This PR tries to improve the command line help text for ``pyani report`` options ``--run_results`` & ``--run_matrices``. Before:

```
$ pyani report --help
...
  --run_results RUN_RESULTS
                        Report table of results for a pyani run (default:
                        False)
  --run_matrices RUN_MATRICES
                        Report matrices of results for a pyani run (default:
                        False)
...
```

After:

```
$ pyani report --help
...
  --run_results RUN_ID  Report table of results for comma separated list of
                        runs (default: False)
  --run_matrices RUN_ID
                        Report matrices of results for comma separated list of
                        runs (default: False)
...
```

This documents that these arguments can be comma separated lists.

The choice of metavariable name is to match ``pyani plot``:

```
$ pyani plot --help
...
  --run_id RUN_ID       run ID to plot (default: None)
...
```

Perhaps the defaults could be consistent - None in both cases?

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
- [X] This is a documentation update

## Action Checklist

- [X] Work on a single issue/concept (if there are multiple separate issues to address, please use a separate pull request for each)
- [X] Fork the `pyani` repository under your own account (please [allow write access for repository maintainers](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork))
- [X] Set up an appropriate development environment (please see `CONTRIBUTING.md`)
- [X] Create a new branch with a short, descriptive name
- [X] Work on this branch
  - [X] style guidelines have been followed
  - [ ] new code is commented, especially in hard-to-understand areas
  - [ ] corresponding changes to documentation have been made
  - [ ] tests for the change have been added that demonstrate the fix or feature works
- [X] Test locally with `pytest -v` **non-passing code will not be merged**
- [X] Rebase against `origin/master`
- [X] Check changes with `flake8` and `black` before submission
- [X] Commit branch
- [X] Submit pull request, describing the content and intent of the pull request
- [ ] Request a code review
- [ ] Continue the discussion at [`Pull requests` section](https://github.com/widdowquinn/pyani/pulls) in the `pyani` repository

### CODE REVIEW

File: pyani/scripts/parsers/report_parser.py [hunk 1]
```diff
@@ -95,29 +95,30 @@ def build(
...
         default=False,
-        help="Report table of all runs in which each genome "
-        + "in the database participates",
+        help="Report table of all runs in which each genome in the database participates",
     )
     parser.add_argument(
         "--run_results",
         action="store",
         dest="run_results",
+        metavar="RUN_ID",
```
Comment (baileythegreen) [inline pyani/scripts/parsers/report_parser.py:111]:
```suggestion
        metavar="RUN_IDS",
```

**Comment by baileythegreen** [comment]
This is good, and potentially means https://github.com/widdowquinn/pyani/issues/332 can be closed, too.

I would suggest changing the metavar to `RUN_IDS`, which still ties in with `pyani plot`, but sounds more like it can accept multiple values.

File: pyani/scripts/parsers/report_parser.py [hunk 2]
```diff
@@ -95,29 +95,30 @@ def build(
...
         default=False,
-        help="Report table of results for a pyani run",
+        help="Report table of results for comma separated list of runs",
     )
     parser.add_argument(
         "--run_matrices",
         action="store",
         dest="run_matrices",
+        metavar="RUN_ID",
```
Comment (baileythegreen) [inline pyani/scripts/parsers/report_parser.py:119]:
```suggestion
        metavar="RUN_IDS",
```

**Review by baileythegreen — COMMENTED** [review]
I think a plural metavar might be clearer.

**Comment by peterjc** [comment]
I did consider ``RUN_IDS`` but that extra character meant an extra line for the command line help. But as you wish?

**Comment by peterjc** [comment]
I think changing the default for these two arguments from ``False`` to ``None`` would help for #332 - shall I do that too?

**Comment by baileythegreen** [comment]
> I think changing the default for these two arguments from `False` to `None` would help for #332 - shall I do that too?

Yes, I think that will clear up a lot of the confusion this causes (me, anyway).

**Comment by baileythegreen** [comment]
This looks good! I'm closing this, and #322. The documentation for `pyani report` will need to be updated, but since that is still in a pending PR of mine, I will handle that.

**Comment by peterjc** [comment]
PS Typo, not issue 3-2-2 but 3-3-2

**Pull request merged**